alerts/monquorum.libsonnet

```diff
@@ -23,7 +23,7 @@
           {
             alert: 'CephMonHighNumberOfLeaderChanges',
             expr: |||
-              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[15m]) > 3
+              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[10m]) > 0
```

Comment:
I feel here you are changing the rules. I would expect these changes to be part of separate PR with proper comments why we need this change.

Reply:
Can we make this into this PR as tests actually add more context and show actual examples where the current alert configuration fail.

Reply:
Ack